tools/pd-ctl/pdctl/command/global.go

```diff
@@ -133,21 +192,23 @@ func postJSON(cmd *cobra.Command, prefix string, input map[string]interface{}) {
 		return
 	}
 
-	url := getAddressFromCmd(cmd, prefix)
-	r, err := dialClient.Post(url, "application/json", bytes.NewBuffer(data))
-	if err != nil {
-		cmd.Println(err)
-		return
-	}
-	defer r.Body.Close()
-
-	if r.StatusCode != http.StatusOK {
-		if err := printResponseError(r); err != nil {
-			cmd.Println(err)
+	tryURLs(cmd, func(endpoint string) error {
+		url := endpoint + "/" + prefix
+		r, err := dialClient.Post(url, "application/json", bytes.NewBuffer(data))
+		if err != nil {
+			return err
+		}
+		defer r.Body.Close()
+		if r.StatusCode != http.StatusOK {
+			if err := printResponseError(r); err != nil {
+				cmd.Println(err)
+			}
+			return nil
```

Comment:
I think in this case, we should try the other URLs?

Reply:
The response error is returned by PD, it is not the error of the PD server,  we should expect to get the same error with all URLs.